Literature/e-Books.md

```diff
@@ -58,7 +58,6 @@ description: Reliable sources to obtain e-Books.
 - [**Bypass Paywalls Clean**](https://gitlab.com/magnolia1234/bypass-paywalls-chrome-clean) - Bypass News Article paywalls.  
 *<small>A more actively maintained fork (with no Google analytics) of [Bypass Paywalls](https://github.com/iamadamdev/bypass-paywalls-chrome).</small>*
 - [**Unpaywall**](https://unpaywall.org/) Browser Extension: [Chromium](https://chrome.google.com/webstore/detail/unpaywall/iplffkdpngmdjhlpjmppncnlhomiipha) & [Firefox](https://addons.mozilla.org/en-GB/firefox/addon/unpaywall/) - When you view a research article, Unpaywall automatically looks for a copy in its index. If it finds one, click the green tab to read the article.
-- [**Outline**](https://outline.com/) - Removes clutter from News Articles.  
```

Comment:
Im planning https://12ft.io/ could be a replacement for this, anyone have any experience with it?

Reply:
yeh its great